src/main/java/com/cloudbees/jenkins/plugins/advisor/casc/AdvisorRootConfigurator.java

```diff
@@ -151,13 +156,17 @@ public CNode describe(AdvisorGlobalConfiguration instance, ConfigurationContext
                         mapping.put(ACCEPT_TOS_ATTR, attribute.describe(instance, context));
                         break;
                     case EMAIL_ATTR:
-                        mapping.put(EMAIL_ATTR, attribute.describe(instance, context));
+                        mapping.put(
+                                EMAIL_ATTR,
+                                String.valueOf(attribute.describe(instance, context)));
```

Comment:
is the `attribute` object honoring the resolver internally?

Reply:
yes, it seems so.  This is also evident in the original issue report.  (Just for fun, i tried adding an explicit encode and you end up with strings like `^^${foo}` :)